rdf/rdf_map.py

```diff
@@ -43,3 +43,31 @@ def write_to_turtle(news_results, country_str):
 
     # write to output file
     g.serialize(destination=f'ttl/{country_str.replace(" ","_")}.ttl', format='turtle')
+
+
+def write_countries_to_turtle(countries):
+    g = Graph()
+    namespace_manager = NamespaceManager(Graph())
+    n_dbpedia_res = Namespace("http://dbpedia.org/resource/")
+    n_schema_res = Namespace("https://schema.org/")
+    n_owl_res = Namespace("http://www.w3.org/2002/07/owl#")
+    n_custom = Namespace("http://www.semanticweb.org/sws/group4/")
+
+    namespace_manager.bind('dbp', n_dbpedia_res, override=False)
+    namespace_manager.bind('schema', n_schema_res, override=False)
+    namespace_manager.bind('sws', n_custom, override=False)
+    namespace_manager.bind('owl', n_owl_res, override=False)
+    g.namespace_manager = namespace_manager
+
+    # country = n_dbpedia_res[country_str.replace(" ", "_")]
+    for country, risk in countries:
+        c = URIRef(n_dbpedia_res[country.replace(" ", "_")])
+
+        # add the country as a named individual
+        g.add((c, RDF.type, n_owl_res.NamedIndividual))
```

Comment:
just because I am curious: why owl:NamedIndividual? does this give any benefit?

Reply:
i just had to import the resource from dbpedia, so i just said it's an individual without any additional information - will be changed due to #2